modules/renter/download.go

```diff
@@ -506,6 +509,7 @@ func (r *Renter) DownloadHistory() []modules.DownloadInfo {
 			EndTime:              d.endTime,
 			Received:             atomic.LoadUint64(&d.atomicDataReceived),
 			StartTime:            d.staticStartTime,
+			StartTimeUnix:        d.staticStartTimeUnix,
```

Comment:
couldn't this be calculated in place, i.e. `StartTimeUnix: d.staticStartTime.UnixNano()`? I don't see the need for a separate variable.

Reply:
yea I can change that